tests/no-dynamic-script-tag.js

```diff
@@ -0,0 +1,38 @@
+const rule = require('../lib/rules/no-dynamic-script-tag')
+const RuleTester = require('eslint').RuleTester
+
+const ruleTester = new RuleTester()
+
+ruleTester.run('no-dynamic-script-tag', rule, {
+  valid: [
+    {
+      code: 'document.createElement("div")'
+    },
+    {
+      code: 'document.createElement("span")'
+    },
+    {
+      code: 'document.createElement("span").type = "foo"'
+    }
+  ],
+  invalid: [
+    {
+      code: 'document.createElement("script")',
+      errors: [
+        {
+          message: "Don't create dynamic script tags, add them in the server template instead.",
+          type: 'Literal'
+        }
+      ]
+    },
+    {
+      code: 'document.createElement("span").type = "text/javascript"',
```

Comment:
This should be a `script` right? Spans don't have a `type`.

Reply:
I just wanted to test that `.type = "text/javascript"` will fail. If I add `document.createElement("script")`, it will fail without checking the `type`